src/s3/document/s3DocumentProvider.ts

```diff
@@ -22,6 +20,11 @@ export class S3DocumentProvider implements vscode.TextDocumentContentProvider {
             showOutputMessage(`${e}`, ext.outputChannel)
             getLogger().error(`S3DocumentProvider: Error opening ${uri.fsPath} on read-only ${e}`)
         }
+
+        if (!data) {
+            getLogger().error(`S3DocumentProvider: Something went wrong opening ${uri.fsPath}`)
```

Comment:
But what went wrong? Can you elaborate?

Reply:
the error is shown on line 21, so I removed this if statement and added a `return ''` on the catch block.